lms/djangoapps/support/static/support/jsx/entitlements/components/Table/EntitlementSupportTable.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { Hyperlink, Table } from '@edx/paragon';
+
+const entitlementColumns = [
+  {
+    label: 'User',
+    key: 'user',
+  },
+  {
+    label: 'Course UUID',
+    key: 'courseUuid',
+  },
+  {
+    label: 'Enrollment',
+    key: 'enrollmentCourseRun',
+  },
+  {
+    label: 'Mode',
+    key: 'mode',
+  },
+  {
+    label: 'Expired At',
+    key: 'expiredAt',
+  },
+  {
+    label: 'Created',
+    key: 'createdAt',
+  },
+  {
+    label: 'Modified',
+    key: 'modifiedAt',
+  },
+  {
+    label: 'Order',
+    key: 'orderNumber',
+  },
+  {
+    label: 'Actions',
+    key: 'button',
+    columnSortable: false,
+    hideHeader: false,
+  },
+];
+
+const parseEntitlementData = (entitlements, ecommerceUrl) =>
+  entitlements.map((entitlement) => {
+    const { expiredAt, created, modified, orderNumber } = entitlement;
+    return Object.assign({}, entitlement, {
+      expiredAt: expiredAt ? moment(expiredAt).format('lll') : '',
+      createdAt: moment(created).format('lll'),
+      modifiedAt: moment(modified).format('lll'),
+      orderNumber: <Hyperlink
+        destination={`${ecommerceUrl}${orderNumber}/`}
+        content={orderNumber || ''}
+      />,
+      button: <div> No Actions Currently Available </div>,
```

Comment:
Does this create a space on both sides of the `div` content? If so, is it for styling purposes? If so, is there a reason not to use CSS?

Reply:
It does not create spaces, html tags strip out excess whitespace (im like 98% sure), not investing much time verifying this, since its just a place holder for now.

Reply:
Just FYI when testing on `stackblitz`

![image](https://user-images.githubusercontent.com/8136030/37470702-78a6fbee-283e-11e8-89c8-772fe79e5ff0.png)

Reply:
Good to know!